internal/file/filesystem.go

```diff
@@ -17,4 +17,4 @@ limitations under the License.
 package file
 
 // ReadWriteByUser defines linux permission to read and write files for the owner user
-const ReadWriteByUser = 0660
+const ReadWriteByUser = 0700
```

Comment:
why are you changing this?

Reply:
Because 0660 is not ReadWriteByUser, is ReadWriteByUserGroup, which makes no sense or required